src/Moq/Times.cs

```diff
@@ -10,14 +10,14 @@ namespace Moq
 	/// <include file='Times.xdoc' path='docs/doc[@for="Times"]/*'/>
 	public struct Times
 	{
-		private Func<int, bool> evaluator;
```

Comment:
This makes a future generic extension point such as #355 impossible. That's however not an issue at all, as `Times` today implements equality comparison, and there is `IOccurrence`, both of which wouldn't work well together with a `Times.Matching(Func<int>)`.